spec/features/editor/editing_entry_outline_spec.rb

```diff
@@ -16,7 +16,7 @@
   scenario 'removing a chapter' do
     user = Dom::Admin::Page.sign_in_as(:editor)
     entry = create(:entry, :title => 'Test Entry', :with_member => user)
-    chapter = create(:chapter, :entry => entry, :title => 'Intro')
+    chapter = create(:chapter, :in_main_storyline_of => entry.draft, :title => 'Intro')
```

Comment:
Useless assignment to variable - `chapter`.<br>Use the new Ruby 1.9 hash syntax.